onboard/teleop/src/__main__.py

```diff
@@ -145,26 +143,58 @@ def drive_control_callback(channel, msg):
         lcm_.publish('/drive_vel_cmd', new_motor.encode())
 
 
-def arm_control_callback(channel, msg):
-    xbox = Xbox.decode(msg)
-    motorSpeeds = [-deadzone(quadratic(xbox.left_js_x), 0.09)*.5,
-                   -deadzone(quadratic(xbox.left_js_y), 0.09)*.5,
-                   quadratic(xbox.left_trigger - xbox.right_trigger)*.60,
-                   deadzone(quadratic(xbox.right_js_y), 0.09)*.75,
-                   deadzone(quadratic(xbox.right_js_x), 0.09)*.75,
-                   (xbox.d_pad_right - xbox.d_pad_left)*0.60,
-                   (xbox.right_bumper - xbox.left_bumper)]
+def ra_control_callback(channel, msg):
+    xboxData = Xbox.decode(msg)
 
-    for i in range(7):
-        openLoopMsg = OpenLoopRAMotor()
-        openLoopMsg.joint_id = i
-        openLoopMsg.speed = motorSpeeds[i]
-        lcm_.publish('/arm_motors', openLoopMsg.encode())
+    motor_speeds = [-deadzone(quadratic(xboxData.left_js_x), 0.09)*0.5,
+                    -deadzone(quadratic(xboxData.left_js_y), 0.09)*0.5,
+                    quadratic(xboxData.left_trigger -
+                              xboxData.right_trigger)*0.6,
+                    deadzone(quadratic(xboxData.right_js_y), 0.09)*0.75,
+                    deadzone(quadratic(xboxData.right_js_x), 0.09)*0.75,
+                    (xboxData.d_pad_right-xboxData.d_pad_left)*0.6]

```

Comment:
The scalars are left over from a few years ago. They should have been removed last year but weren't. Please remove the scalars so that these will be capable of publishing in the full range [-1, 1].  Leave the `0.09` deadzone (I believe that's what that value is), it's probably useful to eliminate noise from bumping the joysticks.